test_main.py

```diff
@@ -0,0 +1,36 @@
+"""
+This is the test !!!
+"""
+import unittest
+from app import get_guess_computer, get_guess_Player, mainFunction
+from unittest.mock import patch
+import io
+import random
+
+class TestMainCode(unittest.TestCase):
+    def test_get_computer(self):
+        with patch('sys.stdout', new= io.StringIO()) as fake_out:
+            result = get_guess_computer()
+            self.assertIsInstance(result, int)
+            self.assertGreaterEqual(result, 1)
+            self.assertLessEqual(result, 100)
+            self.assertEqual(fake_out.getvalue(), '')
```

Comment:
I wouldnt test printing here, in fact, I don't think this function should do any printing

Reply:
all rigth